spec/features/form/postcode_format_spec.rb

```diff
@@ -0,0 +1,38 @@
+require "rails_helper"
+require_relative "helpers"
+
+RSpec.describe "Postcode formatting" do
+  include Helpers
+  let(:user) { FactoryBot.create(:user) }
+  let(:case_log) do
+    FactoryBot.create(
+      :case_log,
+      :in_progress,
+      owning_organisation: user.organisation,
+      managing_organisation: user.organisation,
+    )
+  end
+  let(:id) { case_log.id }
+
+  before do
+    sign_in user
+  end
+
+  context "when a postcode is input" do
```

Comment:
I'm not sure we need feature test for this, we can write a unit test for String class looking into all possible cases and request tests where the postcode formatting will appear on the front.